bumble/gatt.py

```diff
@@ -202,6 +210,13 @@ def __init__(self, uuid, characteristics, primary=True):
         self.characteristics   = characteristics[:]
         self.primary           = primary
 
+    def get_service_advertising_data(self):
+        """
+        Get Service specific advertising data
+        Defined by each Service, default value is empty
```

Comment:
I would make the default `None`, because you could have a service that has service-specific data that's empty (that's legal as per the spec I believe), which is different from a servie that has no service-specific data.

Reply:
ack